pycode/examples/secir_groups.py

```diff
@@ -17,209 +17,158 @@
-    
-    num_groups = contact_polymod.shape[-1]
-    assert num_groups == 8
-    num_compartments = int(InfectionType.Count)
-
-    create_dampings(path='../../data/contact_data/', days=days)
-    dampings = pd.read_csv('../../data/contact_data/dampings.csv', index_col=0, header=[0, 1]).values
-    dampings = np.swapaxes(dampings.reshape(num_groups,days,num_groups),1,2)
-    for i in range(days):
-        dampings[:,:,i] = 0.5*(dampings[:,:,i] + dampings[:,:,i].T)
-
-    sus_ORs=np.array([0.34, 0.67, 1.00, 1.00, 1.00, 1.00, 1.24, 1.47]) # Odds ratios for relative susceptibility -- from https://science.sciencemag.org/content/early/2020/05/04/science.abb8001; 10-20 and 60-70 bins are the average across the ORs
-    symp_probs=np.array([0.50, 0.55, 0.60, 0.65, 0.70, 0.75, 0.80, (0.85 + 0.90)/2])  # Overall probability of developing symptoms (based on https://www.medrxiv.org/content/10.1101/2020.03.24.20043018v1.full.pdf, scaled for overall symptomaticity)
-    severe_probs=np.array([0.00050, 0.00165, 0.00720, 0.02080, 0.03430, 0.07650, 0.13280, (0.20655 + 0.24570)/2]) # Overall probability of developing severe symptoms (derived from Table 1 of https://www.imperial.ac.uk/media/imperial-college/medicine/mrc-gida/2020-03-16-COVID19-Report-9.pdf)
-    crit_probs=np.array([0.00003, 0.00008, 0.00036, 0.00104, 0.00216, 0.00933, 0.03639, (0.08923 + 0.17420)/2]) # Overall probability of developing critical symptoms (derived from Table 1 of https://www.imperial.ac.uk/media/imperial-college/medicine/mrc-gida/2020-03-16-COVID19-Report-9.pdf)
-    death_probs=np.array([0.00002, 0.00006, 0.00030, 0.00080, 0.00150, 0.00600, 0.02200, (0.05100 + 0.09300)/2]) # Overall probability of dying (https://www.imperial.ac.uk/media/imperial-college/medicine/sph/ide/gida-fellowships/Imperial-College-COVID19-NPI-modelling-16-03-2020.pdf)
-
-    # population numbers were taken from
-    # https://de.statista.com/statistik/daten/studie/1365/umfrage/bevoelkerung-deutschlands-nach-altersgruppen/
-    # https://www.bpb.de/nachschlagen/zahlen-und-fakten/soziale-situation-in-deutschland/61538/altersgruppen
-    num_pop = [3.88, 9.71, 12.53, 9.8, 13.7, 11.69, 9.0, 12.4]
-
-    '''probs = Probabilities()
-    probs.set_infection_from_contact(1.0)
-    probs.set_asymp_per_infectious(0.09)  # 0.01-0.16
-    probs.set_risk_from_symptomatic(0.25)  # 0.05-0.5
-    probs.set_hospitalized_per_infectious(0.2)  # 0.1-0.35
-    probs.set_icu_per_hospitalized(0.25)  # 0.15-0.4
-    probs.set_dead_per_icu(0.3)  # 0.15-0.77'''
-
-    # set the params required for the simulation
-    model = SecirModel8()
+    # Define Comartment names
+    compartments = ['Susceptible', 'Exposed', 'Carrier', 'Infected', 'Hospitalized', 'ICU', 'Recovered', 'Dead']
+    # Define age Groups
+    groups = ['0-4', '5-14', '15-34', '35-59', '60-79', '80+']
+    # Define population of age groups
+    populations = [30000, 60000, 150000, 200000, 120000, 100000] #secir simple uses 820000 as pop size, this adds up to 660000
```

Comment:
@lelange Could you please correct this compartment sizes and remove the comment?